### PR TITLE
Created the structure and content for Dart Strings length method

### LINKED ISSUE #4376: [Term Entry] Dart String .length()
### Reason/inspiration (optional)

"We would like a new term entry under Dart [Strings](https://www.codecademy.com/resources/docs/dart/concept/strings) for the Dart Strings `.length()` method. 
Provide the entry in a markdown file under `docs/content/dart/concepts/strings/terms/lenght/length.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - A syntax section with a pseudo code block with an explanation of it.
    - An example showing the method in use.

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by PaulNica**
### Description

- Added a new entry length.md file on the topic "Dart String .length()"
- Closes #4376

### Issue Solved

- Adding a new entry

### Checklist

- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/strings/terms/length/length.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Syntax
+
+```pseudo
+String.length;
+```
+
+## Example
+
+Here is an example of how to use `length` in Dart:
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:28]:
```suggestion
The following example demonstrates the use of `.length` property:
```

File: content/dart/concepts/strings/terms/length/length.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+   // Declaring a string variable called "str"
+   String str = "I love Codecademy";
+   
+   // String length
+   int codecademy = str.length;
+
+   // Print length
+   print(codecademy);
+}
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:41]:
```suggestion
void main() {
  String firstString = "I love Codecademy";
  String secondString = "I love Codecademy😍";

  int lengthOfFirstString = firstString.length;
  int lengthOfSecondString = secondString.length;

  print(lengthOfFirstString);
  print(lengthOfSecondString);
}
```

File: content/dart/concepts/strings/terms/length/length.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+The above code gives the following output:
+
+```shell
+17
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:47]:
```suggestion
17
19
```

File: content/dart/concepts/strings/terms/length/length.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+```
+
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:49]:
```suggestion

> **Note:** The emoji `😍` takes 2 UTF-16 units.

```

File: content/dart/concepts/strings/terms/length/length.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+## Empty string
+
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:51]:
```suggestion
```

File: content/dart/concepts/strings/terms/length/length.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+The **`.length`** property in Dart returns an integer specifying the number of characters in a string (including spaces).
+
+## Syntax
+
+```pseudo
+String.length;
+```
+
+## Example
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:26]:
```suggestion
## Examples
```

File: content/dart/concepts/strings/terms/length/length.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+The above code gives the following output:
+
+```shell
+17
+```
+
+## Empty string
+
+The `length` of an empty string will always be zero:
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:52]:
```suggestion
The following example computes the length of an empty string:
```

File: content/dart/concepts/strings/terms/length/length.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+```dart
+void main() {
+
+   String str = "";
+   int len = str.length;
+   print(len); // Output: 0
+}
+```
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:61]: Add here an output block of the above code.

File: content/dart/concepts/strings/terms/length/length.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+## Empty string
+
+The `length` of an empty string will always be zero:
+
+```dart
+void main() {
+
+   String str = "";
+   int len = str.length;
+   print(len); // Output: 0
```
Comment (letsbuildanything) [inline content/dart/concepts/strings/terms/length/length.md:59]:
```suggestion
   String emptyString = "";
   int lengthOfEmptyString = emptyString.length;
   print(lengthOfEmptyString);
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@PaulNica, Thanks for your great work on this file! 
I've listed some changes below. Could you please review and make them?

**Review by PaulNica — COMMENTED** [review]
Hi @letsbuildanything,

I have made the requested changes. As this is my first-ever contribution I am not sure what the correct way to proceed with the review, so if I did anything wrong or I should've done something different, any feedback would greatly help.

**Comment by letsbuildanything** [comment]
@PaulNica Please commit all the suggestion and add output(```shell) block for the last example as suggested.

**Review by letsbuildanything — APPROVED** [review]
@PaulNica you did a great job! 
It seems ready for next review.

**Review by Sriparno08 — APPROVED** [review]
Looks good! 👍

**Pull request merged**